### DISCUSSION TITLE
No module named 'BB_maps.add_to_cartopy' when testing the demo

### DISCUSSION THREAD

**Discussion by helloqin123** [blaylockbk/goes2go#38] [Q&A] [answered]
Hello, I'm trying to draw the FOV of GOES data. However, the error of 'No module named 'BB_maps.add_to_cartopy'' occurred when testing the 'field-of-view_ABI.ipynb' demo. Could you help me to fix this?

**Comment by blaylockbk** [comment, marked as answer]
Hi @helloqin123,

I'm sorry I haven't kept the docs up to date. `GOES-2-go` is my free-time hobby and there just isn't time to keep everything updated. I would suggest you use the demo notebooks as an example to build on. 

The purpose of that missing function was to plot the data on a Cartopy axes. I have moved and changed a lot about that function here: https://github.com/blaylockbk/Carpenter_Workshop/blob/main/toolbox/cartopy_tools.py.

**Reply by helloqin123** [reply]
It works and really helpful for me, thank you so much!
